Compilation error with new lv2 release (1.18.0)

```
[ 44%] Built target casynth
[ 45%] Building CXX object src/hip2b/CMakeFiles/hip2b_ui.dir/hip2b_ui.cxx.o
cd /tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/hip2b && /gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dhip2b_ui_EXPORTS -I/gnu/store/7pswjcfaq8gx6887gxc1sg4hmyjjpwsl-cairo-1.16.0/include/cairo -I/gnu/store/3vs2gajizp117r3n2sv1xmq3y4kw2f7s-ntk-1.3.1000/include/ntk -I/gnu/store/4nnrb6rpv7n0h0i8gvmwch3gy2n0qvwl-pixman-0.38.4/include/pixman-1 -I/gnu/store/3snpwk7jl8i125bhiilvk9scqc4mnsx7-libpng-1.6.37/include/libpng16 -I/gnu/store/b8pr2k0i2zd07zmb7kpffmcimqi337if-glib-2.60.6/include/glib-2.0 -I/gnu/store/b8pr2k0i2zd07zmb7kpffmcimqi337if-glib-2.60.6/lib/glib-2.0/include -I/gnu/store/n367si248svwbhhmfwsmv74zl98sa0qk-freetype-2.10.1/include/freetype2 -I/gnu/store/xymkwf57x988q8cny2is1dgzrbr9xdfi-util-linux-2.34/include/uuid -I/gnu/store/ain96mrdwqd4s9shdd3s7m4syp5icdx5-libffi-3.2.1/lib/libffi-3.2.1/include -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/hip2b/../ffffltk -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/hip2b/../draw -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/hip2b/. -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/hip2b  -Wall -g -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/hip2b_ui.dir/hip2b_ui.cxx.o -c /tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/hip2b/hip2b_ui.cxx
make[2]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
[ 45%] Built target powerup
[ 47%] Building CXX object src/casynth/CMakeFiles/casynth_ui.dir/casynth_ui.cxx.o
cd /tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/casynth && /gnu/store/x3jx25cd3q363mr7nbgzrhrv1vza6cf7-gcc-7.4.0/bin/c++  -Dcasynth_ui_EXPORTS -I/gnu/store/7pswjcfaq8gx6887gxc1sg4hmyjjpwsl-cairo-1.16.0/include/cairo -I/gnu/store/3vs2gajizp117r3n2sv1xmq3y4kw2f7s-ntk-1.3.1000/include/ntk -I/gnu/store/4nnrb6rpv7n0h0i8gvmwch3gy2n0qvwl-pixman-0.38.4/include/pixman-1 -I/gnu/store/3snpwk7jl8i125bhiilvk9scqc4mnsx7-libpng-1.6.37/include/libpng16 -I/gnu/store/b8pr2k0i2zd07zmb7kpffmcimqi337if-glib-2.60.6/include/glib-2.0 -I/gnu/store/b8pr2k0i2zd07zmb7kpffmcimqi337if-glib-2.60.6/lib/glib-2.0/include -I/gnu/store/n367si248svwbhhmfwsmv74zl98sa0qk-freetype-2.10.1/include/freetype2 -I/gnu/store/xymkwf57x988q8cny2is1dgzrbr9xdfi-util-linux-2.34/include/uuid -I/gnu/store/ain96mrdwqd4s9shdd3s7m4syp5icdx5-libffi-3.2.1/lib/libffi-3.2.1/include -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/casynth/../ffffltk -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/casynth/../draw -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/casynth/. -I/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/casynth  -Wall -g -O2 -g -DNDEBUG -fPIC   -o CMakeFiles/casynth_ui.dir/casynth_ui.cxx.o -c /tmp/guix-build-infamous-plugins-0.2.04.drv-0/build/src/casynth/casynth_ui.cxx
/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/hip2b/hip2b_ui_main.cxx:137:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
 };
 ^
/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/casynth/casynth_ui_main.cxx:185:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
 };
 ^
make[2]: *** [src/hip2b/CMakeFiles/hip2b_ui.dir/build.make:76: src/hip2b/CMakeFiles/hip2b_ui.dir/hip2b_ui_main.cxx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [src/casynth/CMakeFiles/casynth_ui.dir/build.make:76: src/casynth/CMakeFiles/casynth_ui.dir/casynth_ui_main.cxx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/powerup/powerup_ui_main.cxx:134:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
 };
 ^
make[2]: *** [src/powerup/CMakeFiles/powerup_ui.dir/build.make:76: src/powerup/CMakeFiles/powerup_ui.dir/powerup_ui_main.cxx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/tmp/guix-build-infamous-plugins-0.2.04.drv-0/source/src/powercut/powercut_ui_main.cxx:134:1: error: invalid conversion from ‘void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ to ‘void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*) {aka void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)}’ [-fpermissive]
 };
 ^
make[2]: *** [src/powercut/CMakeFiles/powercut_ui.dir/build.make:76: src/powercut/CMakeFiles/powercut_ui.dir/powercut_ui_main.cxx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
make[1]: *** [CMakeFiles/Makefile2:360: src/hip2b/CMakeFiles/hip2b_ui.dir/all] Error 2
make[2]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
make[1]: *** [CMakeFiles/Makefile2:434: src/powerup/CMakeFiles/powerup_ui.dir/all] Error 2
make[2]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
make[1]: *** [CMakeFiles/Makefile2:508: src/powercut/CMakeFiles/powercut_ui.dir/all] Error 2
make[2]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
make[1]: *** [CMakeFiles/Makefile2:183: src/casynth/CMakeFiles/casynth_ui.dir/all] Error 2
make[1]: Leaving directory '/tmp/guix-build-infamous-plugins-0.2.04.drv-0/build'
make: *** [Makefile:133: all] Error 2
command "make" "-j" "8" failed with status 2
```

--- a/src/casynth/casynth_ui_main.cxx
+++ b/src/casynth/casynth_ui_main.cxx
@@ -10,7 +10,7 @@
 
 #define CASYNTHUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#casynth_ui"
 
-static LV2UI_Handle init_casynthUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_casynthUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/cheapdist/cheapdist_ui_main.cxx
+++ b/src/cheapdist/cheapdist_ui_main.cxx
@@ -7,7 +7,7 @@
 
 #define CHEAPDISTUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#cheapdist_ui"
 
-static LV2UI_Handle init_cheapdistUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_cheapdistUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/envfollower/envfollower_ui_main.cxx
+++ b/src/envfollower/envfollower_ui_main.cxx
@@ -12,7 +12,7 @@
 #define ENVFOLLOWERUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#envfollower_ui"
 #define ENVFOLLOWERCVUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#envfollowerCV_ui"
 
-static LV2UI_Handle init_envfollowerUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_envfollowerUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/hip2b/hip2b_ui_main.cxx
+++ b/src/hip2b/hip2b_ui_main.cxx
@@ -7,7 +7,7 @@
 
 #define HIP2BUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#hip2b_ui"
 
-static LV2UI_Handle init_hip2bUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_hip2bUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/lushlife/lushlife_ui_main.cxx
+++ b/src/lushlife/lushlife_ui_main.cxx
@@ -10,7 +10,7 @@
 
 #define LUSHLIFEUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#lushlife_ui"
 
-static LV2UI_Handle init_lushlifeUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_lushlifeUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/powercut/powercut_ui_main.cxx
+++ b/src/powercut/powercut_ui_main.cxx
@@ -9,7 +9,7 @@
 
 #define POWERCUTUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#powercut_ui"
 
-static LV2UI_Handle init_powercutUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_powercutUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/powerup/powerup_ui_main.cxx
+++ b/src/powerup/powerup_ui_main.cxx
@@ -9,7 +9,7 @@
 
 #define POWERUPUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#powerup_ui"
 
-static LV2UI_Handle init_powerupUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_powerupUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

--- a/src/stuck/stuck_ui_main.cxx
+++ b/src/stuck/stuck_ui_main.cxx
@@ -10,7 +10,7 @@
 #define STUCKUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#stuck_ui"
 #define STUCKSTACKERUI_URI "http://ssj71.github.io/infamousPlugins/plugs.html#stuckstacker_ui"
 
-static LV2UI_Handle init_stuckUI(const struct _LV2UI_Descriptor * descriptor,
+static LV2UI_Handle init_stuckUI(const struct LV2UI_Descriptor * descriptor,
 		const char * plugin_uri,
 		const char * bundle_path,
 		LV2UI_Write_Function write_function,

